ErsatzTV/Startup.cs

```diff
@@ -178,6 +183,66 @@ public void ConfigureServices(IServiceCollection services)
                     });
         }
 
+        if (JwtHelper.IsEnabled)
+        {
+            services.AddAuthentication().AddJwtBearer(
+                "jwt",
+                options =>
+                {
+                    options.TokenValidationParameters = new TokenValidationParameters
+                    {
+                        ValidateIssuer = false,
+                        ValidateAudience = false,
+                        ValidateIssuerSigningKey = true,
+                        IssuerSigningKey = JwtHelper.IssuerSigningKey,
+                        ValidateLifetime = true
+                    };
+                    options.Events = new JwtBearerEvents
+                    {
+                        OnMessageReceived = static context =>
+                        {
+                            StringValues token = context.Request.Query["access_token"];
+                            if (!string.IsNullOrWhiteSpace(token))
+                            {
+                                context.Token = token;
+                            }
+
+                            return Task.CompletedTask;
+                        }
+                    };
+                });
+        }
+
+        if (OidcHelper.IsEnabled || JwtHelper.IsEnabled)
+        {
+            services.AddAuthorization(
+                options =>
+                {
+                    if (OidcHelper.IsEnabled)
+                    {
+                        var defaultAuthorizationPolicyBuilder = new AuthorizationPolicyBuilder(
```

Comment:
Will this break if there is no OIDC authentication configured, since it is asking for "cookie" and "oidc" for the default policy?

Reply:
No - it's conditionally added `if (OidcHelper.IsEnabled)`